src/event_loop.rs

```diff
@@ -371,41 +370,51 @@ impl<T> EventLoop<T>
 
                 for event in events.iter() {
                     match event.token() {
-                        CHANNEL => if !self.channel_event(&mut state) {
+                        CHANNEL_TOKEN => if !self.channel_event(CHANNEL_TOKEN, &mut state) {
                             break 'event_loop;
                         },
+
+                        #[cfg(unix)]
+                        token if token == self.pty.child_event_token() => {
+                            while let Some(child_event) = self.pty.get_child_event() {
+                                if child_event == tty::ChildEvent::Exited {
```

Comment:
That works if `Exited` is the only possible event, and currently it is, but I thought it couldn't hurt to be suggestive here.

Reply:
We shouldn't style code for ways things might potentially be in the future.

If there are more events in the future, this can be refactored. Until then a cleaner solution should be preferred.